pkg/controller/config/config_controller.go

```diff
@@ -638,3 +657,19 @@ func requestLogger(req reconcile.Request, context string) logr.Logger {
 		"Request.NamespaceName", req.NamespacedName,
 		"Request.Name", req.Name)
 }
+
+func imagesFromEnv(prefix string) map[string]string {
```

Comment:
shall we move this also to utils, especially if we do
https://github.com/openshift/tektoncd-pipeline-operator/pull/251/files#r388725616

Reply:
Let it be close to this code since its not being used anywhere else.